### PR TITLE
build: use yarn

### PR DESCRIPTION
**Pull request by alan-agius4**
Hopefully this performs better than npm and will enable use to re-enable digest updated.

### CODE REVIEW

**Comment by alan-agius4** [comment]
//cc @mgechev

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,6 +18,17 @@ jobs:
...
+          keys:
+            - yarn-packages-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install Dependencies
+          command: yarn install --frozen-lockfile
+      - save_cache:
+          name: Save Yarn Package Cache
+          key: yarn-packages-{{ checksum "yarn.lock" }}
+          paths:
+            - ~/.cache/yarn
```
Comment (mgechev) [inline .circleci/config.yml]: When does CircleCI caches the packages? Right after installation? Hope we won't have leftovers from ngcc.

Comment (alan-agius4) [reply .circleci/config.yml]: Yes, prior to the build :)

Comment (alan-agius4) [reply .circleci/config.yml]: The steps are in the following order;

- restore cache
- yarn install
- save cache
- build

**Comment by mgechev** [comment]
@alan-agius4 would you resolve the merge conflict so I can move it forward?

**Comment by alan-agius4** [comment]
yes

**Comment by alan-agius4** [comment]
rebases, if you can hold off any more renovate merged until this get in, it will be awesome :)

**Pull request merged**